onnxruntime/test/onnx/main.cc

```diff
@@ -474,7 +474,20 @@ int real_main(int argc, char* argv[], Ort::Env& env) {
     }
 #if !defined(__amd64__) && !defined(_M_AMD64)
     //out of memory
-    static const ORTCHAR_T* x86_disabled_tests[] = {ORT_TSTR("mlperf_ssd_resnet34_1200"), ORT_TSTR("mask_rcnn_keras"), ORT_TSTR("mask_rcnn"), ORT_TSTR("faster_rcnn"), ORT_TSTR("vgg19"), ORT_TSTR("coreml_VGG16_ImageNet")};
+    static const ORTCHAR_T* x86_disabled_tests[] = {ORT_TSTR("BERT_Squad"),
+                                                    ORT_TSTR("bvlc_alexnet"),
+                                                    ORT_TSTR("bvlc_reference_caffenet"),
+                                                    ORT_TSTR("coreml_VGG16_ImageNet"),
+                                                    ORT_TSTR("faster_rcnn"),
```

Comment:
You can leave this file out. It is not used in our CI. They're the outdated code that need be deleted.